kolibri/core/assets/src/state/modules/core/actions.js

```diff
@@ -897,3 +901,38 @@ export function notLoading(store) {
     });
   });
 }
+
+export function fetchUserSyncStatus(store, params) {
+  // for fetching all users that are members of a particular classroom id
+  if (params.member_of) {
+    return UserSyncStatusResource.fetchCollection({
+      force: true,
+      getParams: { member_of: params.member_of },
+    }).then(
+      syncData => {
+        store.commit('SET_CORE_CHANNEL_LIST', _userSyncStatusState(syncData));
```

Comment:
One last question here - why is this being stored using the `SET_CORE_CHANNEL_LIST` mutation? This seems like the wrong place to store this data.

I think we'd be better off not setting any store state in here, and just returning the data from.

Should also return the same data as we are storing if the `_userSyncStatusState` mapping function.